samples/ck-rx65n/basic-sample/src/main.c

```diff
@@ -343,34 +343,60 @@ ULONG   gateway_address = 0;
 #ifndef SAMPLE_DHCP_DISABLE
 static void dhcp_wait()
 {
-ULONG   actual_status;
+    ULONG   actual_status;
+    int status;
 
     printf("DHCP In Progress...\r\n");
 
     /* Create the DHCP instance.  */
-    nx_dhcp_create(&dhcp_0, &ip_0, "DHCP Client");
+    status = nx_dhcp_create(&dhcp_0, &ip_0, "DHCP Client");
+    if(status != NX_SUCCESS) {
+        printf("nx_dhcp_create failed: %d\r\n", status);
+        return;

```

Comment:
I'm not 100% sure about this, but I am concerned about returning from a failure. If there's a problem, nothing will work anyways. It is safrer to stay lock the thread in this function. We can do tx_thread_sleep() in a forever loop. The sleep should also allow printfs to flush.